src/qt/rpcconsole.cpp

```diff
@@ -301,6 +302,14 @@ bool RPCConsole::RPCParseCommandLine(std::string &strResult, const std::string &
                             JSONRPCRequest req;
                             req.params = RPCConvertValues(stack.back()[0], std::vector<std::string>(stack.back().begin() + 1, stack.back().end()));
                             req.strMethod = stack.back()[0];
+#ifdef ENABLE_WALLET
+                            // TODO: Move this logic to WalletModel
+                            if (vpwallets.size() > 1)
+                            {
```

Comment:
I think we should keep the TODO to make sure we clean that up asap (once 0.15 has been split off).